pom.xml

```diff
@@ -117,6 +117,13 @@
             <artifactId>rxjava</artifactId>
             <version>2.2.13</version>
         </dependency>
+        
+   		<dependency>
```

Comment:
Hi @arass! Thanks for the contribution. Seems reasonable.

Can you just line up the indentation with the rest of the file and bump the version up to the latest (2.3.2)?